src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -330,6 +330,14 @@ class InternalAddress: public Address {
 };
 
 class Assembler : public AbstractAssembler {
+public:
+
+  enum class FLOAT_TYPE {
+    half_precision,
+    single_precision,
+    double_precision
+  };
```

Comment:
Seems better to move this enum to file `macroAssembler_riscv.hpp`? It's only used in the macro assember routines.

Reply:
OK.